charts/civil-ccd/values.preview.template.yaml

```diff
@@ -27,28 +27,29 @@ civil-service:
       LAUNCH_DARKLY_ENV: ${SERVICE_FQDN}
       OIDC_ISSUER: https://forgerock-am.service.core-compute-idam-aat2.internal:8443/openam/oauth2/realms/root/realms/hmcts
 idam-pr:
+  debug: true
```

Comment:
This has been added temporarily. Stopping the merge of this PR through review to ensure this is removed before merging.